create_framework/http_kernel_controller_resolver.rst

```diff
@@ -165,15 +165,6 @@ Let's conclude with the new version of our framework::
     use Symfony\Component\HttpKernel;
     use Symfony\Component\Routing;
 
-    function render_template(Request $request): Response
```

Comment:
Can we really remove this function? Where would it be loaded then?

Reply:
> To celebrate the birth of our new framework, let's create a brand new
application that needs some simple logic. Our application has one page that
says whether a given year is a leap year or not

After this part we do not use `render_template` function in created Controllers